tests/TestCase/Shell/PhpstormShellTest.php

```diff
@@ -67,6 +67,26 @@ public function tearDown() {
 	/**
 	 * @return void
 	 */
+	public function testDirExists()
```

Comment:
You are actually testing the FileExists not DirExists here

Reply:
file_exists check for both file and dir : http://php.net/manual/en/function.file-exists.php

Reply:
I think we'll need to add a dir in TMP to put the file in